packages/next-international/src/app/client/create-i18n-provider-client.tsx

```diff
@@ -3,30 +3,30 @@ import type { Context, ReactNode } from 'react';
 import React, { Suspense, use, useMemo } from 'react';
 import { flattenLocale } from '../../common/flatten-locale';
 import type { LocaleContext } from '../../types';
-
```

Comment:
Could you revert the deletion of empty lines? It makes it harder to read the code.

Reply:
I'm sorry, it seems the formatter ran automatically. I'll fix it.

Reply:
fixed by https://github.com/QuiiBz/next-international/pull/252/commits/860e13c0a3536748381eeeebbd938fba6ea0b6ad